Makefile

```diff
@@ -1,11 +1,11 @@
 PROJECT = rabbitmq_lvc_exchange
 PROJECT_DESCRIPTION = RabbitMQ Last Value Cache exchange
 
-RABBITMQ_VERSION ?= v3.13.x
+RABBITMQ_VERSION ?= v4.0.x
```

Comment:
I suggest to bump to 4.1 directly, so that core team can accept more easily (they probably wont release a version for 4.0)